### PR TITLE
Chore: Rename references to sns aggregator

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

SNS Caching is a misleading name. Rename all references to SNS Aggregator.

# Changes

* Renaming of variables, env vars and files.

# Tests

* Fix tests after renaming.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@mstrasinskis @peterpeterparker please review

File: dfx.json [hunk 1]
```diff
@@ -200,7 +200,7 @@
         "FEATURE_FLAGS": {
           "ENABLE_SNS_2": true,
           "ENABLE_SNS_VOTING": true,
-          "ENABLE_SNS_CACHING": true,
+          "ENABLE_SNS_AGGREGATOR": false,
```
Comment (peterpeterparker) [inline dfx.json:203]: what few of these becomes `false`?

Comment (lmuntaner) [reply dfx.json:203]: All of them. They are not yet supported and I preferred to do like you did for `ENABLE_CKBTC_LEDGER`. By default, disabled.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**